src/Angular.js

```diff
@@ -232,8 +232,7 @@ function isArrayLike(obj) {
 
   // NodeList objects (with `item` method) and
   // other objects with suitable length characteristics are array-like
-  return isNumber(length) &&
-    (length >= 0 && ((length - 1) in obj || obj instanceof Array) || typeof obj.item === 'function');
+  return isNumber(length) && (length >= 0 && (length - 1) in obj || typeof obj.item === 'function');
```

Comment:
I know. But previous there was also the restriction that `isNumber(length)` and `length >= 0`, which is gone now.

What I am saying is that `isArrayLike` (and anything depending on it) is slightly affected by the change in `isArray`.